egs/voxceleb/v1/local/make_voxceleb1_v2.pl

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/perl
+#
+# Copyright 2018  Ewald Enzinger
+#           2018  David Snyder
+#           2019  Soonshin Seo
+#
+# Usage: make_voxceleb1.pl /export/voxceleb1 data/
```

Comment:
Sorry to nitpick, could you fix this comment, the usage is for the old script, not the new one.
Also, while you're here, could you write a comment that briefly summarizes why we need this script. Just say what you said before in the run.sh. Something like "The VoxCeleb 1 corpus underwent several updates that changed the directory and speaker ID format. The script make_voxceleb1.pl works for the oldest version of the corpus. This script should be used if you've downloaded the corpus recently."

Reply:
Fine. I agree with your opinion. I think it is an important point. 
Thanks.